### PR TITLE
slow queries

### PR DESCRIPTION
**Pull request by rfair404**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -258,6 +258,12 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** Where do slow queries go to retire?
+
+**A:** A little cabin by the latency.
```
Comment (styfle) [inline readme.md]: I don't get it? Is there a Laten Sea or something?

Comment (rfair404) [reply readme.md]: not that I know of.

Comment (rfair404) [reply readme.md]: I re-worked it a bit, see if that makes more sense.

**Review by styfle — APPROVED** [review]
🐚

**Pull request merged**